### ISSUE TITLE
How do I reduce extreme verbosity?

### ISSUE THREAD

**Issue by matthew-dean** [samchon/typia#866]
## Question
I have a number of functions for which I want to validate their parameters. For example, I have a function like this:

```ts
function floor(value: Dimension) {
  validate(typia.validate<Parameters<typeof floor>>(arguments), floor)
  return new Dimension([Math.floor(value.number), value.unit])
}
```
Now, what I want is to simply validate that each argument is the supplied type, and then manage the errors in the `validate()` function. However, `typia` generates a huge number of irrelevant tests (this 4-line function turns into 1700 lines generated by Typia, a non-starter), by traversing into the `Dimension` prototype (which extends `Node`) and generate tests for the base (`Node`) type, validating every property (??).

I literally just want something like `value instanceof Dimension`. However, even the `is` function does not do this, even though it's supposedly just to test an instance type.

The reason why I'm not manually just writing a `value instanceof Dimension` and using Typia is because:
1. There are a large number of functions to do argument testing with,
2. Some functions might have overload signatures, which makes it a little more complex to abstract.

I tried looking at the `createValidate` function, but I really don't see how to use this to reduce the generated overhead.

**Note:** even if I restrict the type to something like:
```ts
type Arguments<T> = {
  [K in keyof T]: T[K] extends Node ? { type: T[K]['type'] } : never
}

function floor(value: Dimension) {
  validate(typia.validate<Arguments<Parameters<typeof floor>>>(arguments), floor)
  return new Dimension([Math.floor(value.number), value.unit])
}
```
... in other words, just validating that it has a type property that is the same as the passed in object, the Typia generated code is really still exceptional in length. I understand that the reason for this is that there are probably a number of type edge cases that have to be accounted for, but is there a way to customize this and dramatically reduce output?

**Comment by samchon** [comment]
The reason why not skipping only with `instanceof` statement is, one of its properties can protobit type.

I'm sorry but no way in the case. What about using `Omit` type properly?

```type
typia.assert<Omit<T, keyof U>>(input);
```

**Comment by matthew-dean** [comment]
@samchon I don't understand your answer. I think ultimately my question is how to avoid Typia generating 1,000+ lines of validation code.

**Comment by samchon** [comment]
`typia` validates in the primitive type level. It does not perform `instanceof` statement.

Also, you're saying that the `instanceof` satement can skip the 1,000+ lines of validation codes, but it is not true. Even though the class type is matched, how you can ensure its properties (members) are correct?
